projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -1715,28 +1803,28 @@ public function render_amp_reader_mode_css() {
 	 * Sets up the shortcode processing.
 	 *
 	 * @uses add_filter, add_shortcode
-	 * @return null
 	 */
-	protected function _setup_shortcode() {
+	protected function setup_shortcode() {
 		add_filter( 'the_content', array( $this, 'test_for_shortcode' ), 0 );
 
 		add_shortcode( self::SHORTCODE, array( $this, 'get_client_rendered_html' ) );
 	}
 
-	protected function _allow_feature_toggle() {
-		if ( null === $this->_allow_feature_toggle ) {
-			/**
-			 * Filter the display of the Related Posts toggle in Settings > Reading.
-			 *
-			 * @module related-posts
-			 *
-			 * @since 2.8.0
-			 *
-			 * @param bool false Display a feature toggle. Default to false.
-			 */
-			$this->_allow_feature_toggle = apply_filters( 'jetpack_relatedposts_filter_allow_feature_toggle', false );
+	/**
+	 * Filter the display of the Related Posts toggle in Settings > Reading.
+	 *
+	 * @module related-posts
+	 *
+	 * @since 2.8.0
+	 *
+	 * @return bool false Display a feature toggle. Default to false.
+	 */
+	protected function allow_feature_toggle() {
+		if ( null === $this->allow_feature_toggle ) {
+
+			$this->allow_feature_toggle = apply_filters( 'jetpack_relatedposts_filter_allow_feature_toggle', false );
 		}
```

Comment:
this is problematic since we're losing the docblock for the filter; can we document both the method and the filter?